npm/npm_import.bzl

```diff
@@ -347,16 +423,11 @@ def npm_translate_lock(
             }
             ```
 
+            Dicts of bins not additive. The move specific name match takes precedence.
```

Comment:
do we have a mix of these where some are additive and some are most-specific-wins?

Reply:
Yes. Depending on the attribute one makes more sense than the other

Reply:
can you make all of them indicate Additive or Most-Specific in a consistent way? seems like this will be confusing

Reply:
I have,

```
Foo are not additive. The move specific name match takes precedence.
```

and

```
Bar are additive when there are multiple matches for a package. More specific matches are
appended to previous matches.
```

You have something better in mind?